crates/ra_syntax/src/syntax_error.rs

```diff
@@ -126,31 +126,31 @@ impl fmt::Display for TokenizeError {
             TokenizeError::EmptyInt => "Missing digits after integer base prefix",
             TokenizeError::EmptyExponent => "Missing digits after the exponent symbol",
             TokenizeError::UnterminatedBlockComment => {
-                "Missing trailing `*/` symbols to terminate the block comment"
+                "Missing trailing */ symbols to terminate the block comment"
             }
             TokenizeError::UnterminatedChar => {
-                "Missing trailing `'` symbol to terminate the character literal"
+                "Missing trailing ' symbol to terminate the character literal"
             }
             TokenizeError::UnterminatedByte => {
-                "Missing trailing `'` symbol to terminate the byte literal"
+                "Missing trailing ' symbol to terminate the byte literal"
             }
             TokenizeError::UnterminatedString => {
-                "Missing trailing `\"` symbol to terminate the string literal"
+                "Missing trailing \" symbol to terminate the string literal"
             }
             TokenizeError::UnterminatedByteString => {
-                "Missing trailing `\"` symbol to terminate the byte string literal"
+                "Missing trailing \" symbol to terminate the byte string literal"
             }
             TokenizeError::UnterminatedRawString => {
-                "Missing trailing `\"` with `#` symbols to terminate the raw string literal"
+                "Missing trailing \" with # symbols to terminate the raw string literal"
             }
             TokenizeError::UnterminatedRawByteString => {
-                "Missing trailing `\"` with `#` symbols to terminate the raw byte string literal"
+                "Missing trailing \" with # symbols to terminate the raw byte string literal"
             }
             TokenizeError::UnstartedRawString => {
-                "Missing `\"` symbol after `#` symbols to begin the raw string literal"
+                "Missing \" symbol after # symbols to begin the raw string literal"
```

Comment:
I usually like to place some kind of quotes around snippets because that's helpful when a snippet contains trailing/leading whitespace or other invisible characters. But we should just stick to what rustc is doing.

Which... I am not sure what is exactly! For this code

```rust
λ cat main.rs 
fn main() {
    let s = r###;
}
```

I get this error

![image](https://user-images.githubusercontent.com/1711539/73169966-980b4380-40fd-11ea-9d5f-a4e3ddec444d.png)

Note how `#` is uselessly in quotes, but the empty string between `:` and  `;` isn't.

@estebank what are the current guidelines around quotes?

Reply:
Heh, I initially wrapped the symbols into backticks (hoping they would be rendered as Markdown), but once I started VSCode and checked out that the messages were displayed as a raw text I removed them in this commit:
https://github.com/rust-analyzer/rust-analyzer/pull/2911/commits/34d72aa0c9bb73a2dbfa3ed6dedd5fe88e4aaed7

Reply:
We try to consistently surround code and tokens (including identifiers) in backticks. No tool is actually _parsing_ them, but having them bare makes it harder to read (consider <pre>found , but expected .</pre> vs <pre>found \`,\` but expected \`.\`</pre>)